src/PiWeb.Import.Sdk/ConfigurationItems/PartSelectionConfigurationItem.cs

```diff
@@ -0,0 +1,85 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.ConfigurationItems;
+
+using System;
+using PropertyStorage;
+
+/// <summary>
+/// Represents a configuration item for a property pointing to a specific part.
+/// </summary>
+public class PartSelectionConfigurationItem : ConfigurationItemBase
+{
+	#region members
+
+	private string _Value;
+
+	#endregion
+
+	#region constructors
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="PartSelectionConfigurationItem"/> class.
+	/// </summary>
+	/// <param name="defaultValue">The default value.</param>
+	public PartSelectionConfigurationItem( string defaultValue = "/" )
+	{
+		_Value = defaultValue;
+	}
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="PartSelectionConfigurationItem"/> class.
+	/// Associates this configuration item to a property of the given <paramref name="storage"/>.
+	/// </summary>
+	/// <param name="storage">The property storage to read from and write to.</param>
+	/// <param name="key">The key of the property to associate the configuration item with.</param>
+	/// <param name="defaultValue">The default value that is used if the property does not exist in the storage.</param>
+	public PartSelectionConfigurationItem( IPropertyStorage storage, string key, string defaultValue )

```

Comment:
How should we handle empty string? Empty string is not a valid path in the inspection plan.
Should we simply treat it as "/" (_path to root part_) or leave it as invalid?

Reply:
I think we should treat it as invalid since the corresponding api methods to convert to PathInformation will not accept it either. We should, however, make sure the standard value for this new configuration item (if not specified) is something else. Probably the root part.